### ISSUE TITLE
Empty links

### ISSUE THREAD

**Issue by phucduong86** [phucduong86/phucduong86.github.io#6]
Icons on the timeline column doesn't have any text associated with it.
Maybe it doesn't need a link at all? Seems redundant.

**Comment by phucduong86** [comment]
![2018-11-07_1541](https://user-images.githubusercontent.com/21692768/48159646-ce179a00-e2a3-11e8-8b9d-2c95cc9e98dd.png)
